.gitbook/overview/roadmap.md

```diff
@@ -22,22 +22,22 @@
 - Enhance long-term sustainability of validator ecosystem
 - Improve governance process
 
-### 2022 Q3: Data Market Protocol v0 on Testnet
+### 2022 Q3: Data Exchange Protocol v0 on Testnet
 
-- Evaluate data market protocol v0 on Panacea Testnet
+- Evaluate data exchange protocol v0 on Panacea Testnet
   - Technical evaluation
   - Product-wise evaluation
 - Revise whitepaper
```

Comment:
```suggestion
- Update and release new version of whitepaper
```

Reply:
thank you.
done: https://github.com/medibloc/panacea-core/commit/1c13ab2c997ee2016c7be4a194006feef9f8be98